pkg/trie/triedb/codec/decode_test.go

```diff
@@ -0,0 +1,438 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package codec
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func scaleEncodeBytes(t *testing.T, b ...byte) (encoded []byte) {
```

Comment:
Is it necessary to have both these helpers?

Reply:
We are using for the `Decode` test